scripts/main.js

```diff
@@ -52,11 +52,11 @@ module.exports = (robot) => {
         }
     })
     robot.error((err, res) => {
-        console.log(err)
-        robot.logger.error("DOES NOT COMPUTE")
+        console.log(err.message)
+        robot.logger.error(err.message)
```

Comment:
@zelima Can we keep error mesages as it was before, `DOES NOT COMPUTE`. I read official documentation, and could not find err.message property, and according to their documentation, it says "You can do anything you want here, but you will want to take extra precaution of rescuing and logging errors, particularly with asynchronous code. Otherwise, you might find yourself with recursive errors and not know what is going on."